config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
+
+  resources :employee
+  resources :assignment
+  resources :week
+
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  #get  '/schedule', to: 'assignment#index'
+  #get  '/employee', to: 'employee#index'
+  post '/employee/new', to: 'employee#create'
```

Comment:
We should remove this. Let me know if it's giving you troubles. The conventional route here (covered by line 3 above) is `POST /employees`.